src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -144,7 +182,7 @@ struct Property : public util::BaseProperty<T, mutability_> {
      */
     template <typename... Args>
     inline std::pair<std::string, Any> operator()(Args&&... args) const {
-        return {this->name(), Any::make<T>(std::forward<Args>(args)...)};
+        return {this->name(), Any::make<T>(Forward<Args>{std::forward<Args>(args)}...)};
```

Comment:
why do we need custom forward?

Reply:
To handle string input to not string type.